src/coreclr/jit/jitconfigvalues.h

```diff
@@ -256,6 +256,11 @@ CONFIG_INTEGER(EnableIncompleteISAClass, W("EnableIncompleteISAClass"), 0) // En
 CONFIG_METHODSET(JitDisasm, W("JitDisasm"))
 #endif // !defined(DEBUG)
 
+// Print raw bytes for instructions, e.g.
+//
+//   488B09               mov      rcx, gword ptr [rcx]
+CONFIG_INTEGER(JitDisasmShowRawInstructions, W("JitDisasmShowRawInstructions"), 0)
```

Comment:
I'd suggest naming this `JitDisasmWithEncoding` or `JitDisasmWithBytes`.

Reply:
I like `JitDisasmWithEncoding`.

Reply:
So, this will be breaking change compared to what we had in .NET 7.